adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Platform/PlatformInformationBase.cs

```diff
@@ -74,19 +58,13 @@ public virtual bool GetCacheLoadPolicy(IPlatformParameters parameters)
             return true;
         }
 
-        public virtual Uri ValidateRedirectUri(Uri redirectUri, RequestContext requestContext)
+        public override void ValidateRedirectUri(Uri redirectUri, RequestContext requestContext)
```

Comment:
>void [](start = 24, length = 4)

IMO: this should still return the URI, now the caller has to know nothing is done to the input param. Alternatively the method should change name: ThrowIfRedirectUriIsNotValid ??? Then it is more obvious what the method does and that the flow will be terminated in case something fails.

Reply:
Or "ThrowIfRedirectUriIsNull" as this is just checking that the redirectUri is not null

---
In reply to: [215738440](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1244#discussion_r215738440) [](ancestors = 215738440)